src/client.h

```diff
@@ -557,6 +557,11 @@ class Client : public con::PeerHandler, public InventoryManager, public IGameDef
 
 	void makeScreenshot(IrrlichtDevice *device);
 
+	void displayChat(const std::wstring &input)
```

Comment:
I suggest calling this pushToChatLog or similar
Maybe this should also be inlined and used in handlecommand_ChatMessage